a/appstream/stone.yml

```diff
@@ -0,0 +1,45 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : appstream
+version     : '1.0.1'
+release     : 1
+homepage    : https://www.freedesktop.org/wiki/Distributions/AppStream/
+upstreams   :
+    - https://www.freedesktop.org/software/appstream/releases/AppStream-1.0.1.tar.xz : fa4d5994e5c6f22e9b66cb0b52e8b5f5d96e7353ea5528b56436e0150829203b
+summary     : Tools and libraries to work with AppStream metadata
+description : |
+    AppStream is a cross-distribution effort for enhancing the way we
+    interact with the software repositories provided by (Linux) distributions
+    by standardizing software component metadata.
+license     :
+    - LGPL-2.1-or-later
+builddeps   :
+    # TODO: SVG, steamming, docs
+    - docbook
+    - binary(cmake)
+    - binary(gperf)
+    - binary(itstool)
+    - binary(msgfmt)
+    - binary(vapigen)
+    - binary(xsltproc)
+    - pkgconfig(gio-2.0)
+    - pkgconfig(glib-2.0)
+    - pkgconfig(gobject-2.0)
+    - pkgconfig(gobject-introspection-1.0)
+    - pkgconfig(libcurl)
+    - pkgconfig(libsystemd)
+    - pkgconfig(libxml-2.0)
+    - pkgconfig(libzstd)
+    - pkgconfig(xmlb)
+    - pkgconfig(yaml-0.1)
+toolchain   : gnu
+setup       : |
+    sed -i '/missing-include-dirs/d' meson.build
+    %meson -Dstemming=false -Dvapi=true -Dsvg-support=false -Dapidocs=false -Dinstall-docs=false --sysconfdir=/usr/share/appstream
```

Comment:
Please use the %(datadir) definition (defined here: https://github.com/serpent-os/boulder/blob/main/data/macros/arch/base.yml#L11)

Reply:
Fixed!